tachyon/math/polynomials/multivariate/linear_combination_term.h

```diff
@@ -47,7 +47,7 @@ struct LinearCombinationTerm {
       absl::Span<const size_t> indexes) {
     return std::accumulate(
         indexes.begin(), indexes.end(), F::One(),
-        [&flattened_ml_evaluations, &point](F& acc, const size_t index) {
+        [&flattened_ml_evaluations, &point](F& acc, size_t index) {
```

Comment:
I was talking with @dongchangYoo about how `size_t` does not need to be `const` as a local variable and thought to change it, yet since theres no reason to change code I've already written that way, I'll change back that change.